elastic/kibana
# [Graph] Fix drilldown url templates for sample data

Closes https://github.com/elastic/kibana/issues/72541

## Summary

This PR fixes url templates which are used to create drilldowns in Graph for sample data.

![Sep-20-2022 18-14-27](https://user-images.githubusercontent.com/1415710/191310696-b601bf9b-fa0c-4504-b7f0-e340eb5093b8.gif)

Testing:
- Install sample data and navigate to the created Graph workspace. Please test drilldowns for graphs in the default and a custom spaces (it should work now, before was "Not found" page)
- Create a new Graph workspace and check drilldowns there too (should work as before)
- Try editing a Graph workspace, saving it and check the drilldowns.
